docs/UserGuide.md

```diff
@@ -336,6 +364,10 @@ Restrictions:
 * The event id must be positive and must correspond to exactly one of the ids of the events currently listed.
 * The volunteer id must be positive and must correspond to exactly one of the ids of the volunteers currently listed.
 * The volunteer must not already be added to the event.
+* The number of volunteers within the event should not be equal to the maximum number of volunteers allowed for the event, since adding a new volunteer
+  at that point would result in the number of volunteers exceeding the maximum allowed. 
+  * **Note:** To change the limit on the maximum number of volunteers for a certain event, use the
```

Comment:
consider using tips instead of note for standardization

Reply:
not standardisation, it's just cos it's a tip lol good idea